eng/jacoco-test-coverage/pom.xml

```diff
@@ -80,7 +80,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-identity;current} -->
+      <version>1.0.1</version> <!-- {x-version-update;com.azure:azure-identity;current} -->
```

Comment:
Is this meant to be a patch or minor version bump?

Reply:
We spoke in standup yesterday  and we are good with 1.0.1 version.